src/vivarium_gates_nutrition_optimization/components/hemoglobin.py

```diff
@@ -264,15 +262,21 @@ def adjust_hemoglobin_exposure(
         return hemoglobin_exposure
 
 
-class Anemia:
-    @property
-    def name(self):
-        return "anemia"
-
+class Anemia(Component):
     @property
     def columns_created(self):
         return ["anemia_status_at_birth"]
 
+    # TODO MIC-4366: We include tracked here as a bandaid, given new RMS essentially requires
```

Comment:
So this a change we'd like to make for every pipeline that gets used by an observer?

Reply:
I would say yes, though this also only matters for models where simulants are becoming untracked through some other mechanism than dying (e.g. aging out). I **think** this should generally fail pretty obviously when it is a concern, but as is often the case with stuff like that, I'm not 100% certain.